fabric/README.md

```diff
@@ -0,0 +1,18 @@
+<!---
+Licensed under Creative Commons Attribution 4.0 International License
+https://creativecommons.org/licenses/by/4.0/
+--->
+
+# Build Fabric with FPC support
+
+Make sure you have built [tlcc](../tlcc),[tlcc_enclave](../tlcc_enclave), and [plugins](../plugins)
```

Comment:
Maybe mention that a top-level build is supposed to do this automatically. If we think have to do this manual, it might be worth adding a target for above list?

Reply:
actually we only need to build `tlcc_enclave`. Updated